doc/ruleSets/PER-CS2.0.rst

```diff
@@ -0,0 +1,10 @@
+=======================
+Rule set ``@PER-CS2.0``
+=======================
+
+EXPERIMENTAL: Rules that follow `PER Coding Style 2.0 <https://www.php-fig.org/per/coding-style/>`_.
```

Comment:
*question*

why Experimental?
what is experimental here?

Reply:
If I see right, no style changed, v2 only adds style description for new language syntax
ref https://github.com/php-fig/per-coding-style/compare/1.0.0...master#diff-bc6661da34ecae62fbe724bb93fd69b91a7f81143f2683a81163231de7e3b545R4

Reply:
It's because first item on the checklist in #7247 - we just need to let people know that ruleset is not a final version of PER-CS v2.

Reply:
> It's because first item on the checklist in https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/7247

it's not answering my question, actually ;)

> we just need to let people know that ruleset is not a final version of PER-CS v2.

none rulset is considered final and each ruleset can evolve into future, they are not locked and not modified ever.
2 best representative examples:
- PSR12 ruleset is present, yet not reflecting _each_ rule described in PSR spec. amount of rules will increase over time, covering more and more aspects of PSR spec
- Symfony ruleset is present, yet standard is evolving over time, so also the ruleset is getting adjusted

with that, we never considered a rulset to be "experimental/internal" because "it's not complete".

with above, please remove the experimental marker. using this ruleset has nothing to do with experiments.

Reply:
It IS answering, @Crell did it because I asked for it. Whether that makes sense it's totally different story 😉.

Reply:
i got you asked for it, but still that part didn't resolved my question :sweat:
 later part did, thanks

Reply:
TL;DR, @Crell , please remove the experimental marker